app/assets/stylesheets/utils/_variables.scss

```diff
@@ -1,5 +1,8 @@
 // Colors
-
+$yellowFF: #FFCD1C;
+$yellowFAF: #FAFF00;
+$graye1: #E1E1E1;
```

Comment:
Preferably use variable names that are more meaningful, for example: primary, secondary, disabled, enabled, page_background, title, subtitle, etc.